test/functional/rpc_net.py

```
@@ -7,11 +7,18 @@
+from test_framework.blocktools import COINBASE_MATURITY
 import test_framework.messages
 from test_framework.messages import (
     MAX_PROTOCOL_MESSAGE_LENGTH,
-    NODE_NETWORK,
+)
+from test_framework.p2p import (
+    P2PInterface,
+    P2P_SERVICES,
 )
 
 from itertools import product
```

Ruff (0.12.2)

Redefinition of unused `product` from line 11

Remove definition: `product`

(F811)